Automattic/crowdsignal-forms
# Show branding on editor

This PR displays branding links while editing the poll/vote/applause blocks in order to show a consistent preview.

Brand link should now show exactly how it will be seen on the public post. A tooltip has been added on mouseover to invite the user to check the upgrade options:
![image](https://user-images.githubusercontent.com/157240/102114479-0261cb00-3e19-11eb-9f45-8e5d60d8b34c.png)


## Test instructions:

NOTE: this PR depends on D54527-code which hasn't been deployed yet.

Checkout and run `make client`. With a free account, create/edit a post with a Crowdsignal Form's block. Shortly after inserting the block, the brand link should be viewable.
